pod2man.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```

Comment:
Not sure this needs to be *super* portable, but if so, this can help `#!/usr/bin/env bash`.